install-deps.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash -e
+#!/bin/sh -e
```

Comment:
On 4-2-2016 19:41, Sage Weil wrote:
> In install-deps.sh
> <https://github.com/ceph/ceph/pull/7515#discussion_r51917069>:
> 
>> @@ -1,4 +1,4 @@
>> -#!/bin/bash -e
>> +#!/bin/sh -e
> 
> This breaks when sh == dash (which it is on ubuntu):

Ah, right, I've ran into this long time ago.
So I should have known this.

Problem is that on FreeBSD bash doesn't live in /bin. :(
One reason I have a softlink to /us/local/bin/bash for this.

not sure if
#!/usr/bin/env bash
is a sollution. And would this perl script type hack work in these cases
on Linux?

Otherwise it will require require some extra softlinking before
install-deps.sh can be executed. Also note that install-deps.sh will not
run on a fresh FreeBSD if the bash-package is not installed.

I can fix all this when do_freebsd.sh is the startup entry-point for
FreeBSD builds, and do a install-deps-freebsd.sh from there. Would that
be acceptable?

> 
> ./install-deps.sh: 126: ./install-deps.sh: Syntax error: "(" unexpected

'mmm, at first glance I can't seem to point the error.
And the Jenkins run did not show it either, but perhaps it does not run
install-deps.sh

But I'll run it in my Centos7 VM and see what happens.
Otherwise wen I take the FreeBSD code out of install-deps.sh things
should return to normal.

--WjW